fact-bounty-client/src/shared/OAuthContainer/FacebookContainer.js

```diff
@@ -15,13 +17,14 @@ class FacebookContainer extends React.Component {
   }
 
   responseFacebook = res => {
+    console.log(res);
```

Comment:
@Tushar19varshney Done. Also can you write migration for [this](https://github.com/scorelab/fact-Bounty/pull/219/files#diff-ea82e5e1b60fc5fcc95a9f383f904263L24) change in pull request #219. Presently new user entry from social login is not being saved in the database because the password cannot be empty.